lib/pdk/module/update.rb

```diff
@@ -33,19 +33,15 @@ def run
           return unless PDK::CLI::Util.prompt_for_yes(message)
         end
 
-        # Remove these files straight away as these changes are not something that the user needs to review.
-        if needs_bundle_update?
-          update_manager.unlink_file('Gemfile.lock')
-          update_manager.unlink_file(File.join('.bundle', 'config'))
-        end
+        # Remove these files straight away as these changes are not something
+        # that the user needs to review.
+        update_manager.unlink_file('Gemfile.lock')
+        update_manager.unlink_file(File.join('.bundle', 'config'))
 
         update_manager.sync_changes!
 
-        if needs_bundle_update?
-          require 'pdk/util/bundler'
-
-          PDK::Util::Bundler.ensure_bundle!
-        end
+        require 'pdk/util/bundler'
+        PDK::Util::Bundler.ensure_bundle!
```

Comment:
Should this be a user prompt?

`Would you like to update your bundle?`

Reply:
No, I don't think so. If it doesn't update, then it'll just happen automatically when they run the next PDK command.

Reply:
Only usecase I can think of is bundle taking FOREVER (looking at you Windows) and wanting to stop that for tools like `pdksync`.  But I'm more than happy to add that later if needs be.